tests/brmsfit_examples.R

```diff
@@ -84,16 +84,24 @@ brmsfit_example6 <- brm(
   stan_model_args = stan_model_args, rename = FALSE
 )
 
+brmsfit_example7 <- brm(
```

Comment:
These internal models take quite some space so I would like to avoid saving another one. Can you add this test to the local tests in `tests/local/`? Perhaps there is already a censored model.

Reply:
Good idea thanks, have reverted those changes and moved. Though when running the local tests beforehand I noticed the `add_rstan_model` function fails for that kidney survival example. Is this the same on your computer, and if so, is it okay to ignore for this PR and handle separately?

Test fail:
```
-- Error (tests.models_new.R:89:3): Survival model from brm doc works correctly ---------------------------
Error in `stanc(file = file, model_code = model_code, model_name = model_name, 
    verbose = verbose, obfuscate_model_name = obfuscate_model_name, 
    allow_undefined = allow_undefined, isystem = isystem)`: failed to parse Stan model '530b15df4662e9b168e0f021510d8a97' due to the above error.
Backtrace:
 1. brms::add_rstan_model(fit3)
      at tests.models_new.R:89:2
 4. rstan::stan(...)
      at brms/R/brmsfit-helpers.R:761:4
 5. rstan::stan_model(...)
 6. rstan::stanc(...)
```

Checking internally looks like the arrays from the `cmdstanr` backend:
```
rstan::stan(
+       model_code = stancode(x, threads = threading()),
+       data = standata(x), chains = 0
+     )
SYNTAX ERROR, MESSAGE(S) FROM PARSER:
 error in 'modeldb4faf53a2_530b15df4662e9b168e0f021510d8a97' at line 8, column 2
  -------------------------------------------------
     6:   int<lower=1> N; // total number of observations
     7:   vector[N] Y; // response variable
     8:   array[N] int<lower=-1, upper=2> cens; // indicates censoring
         ^
     9:   int<lower=1> K; // number of population-level effects
  -------------------------------------------------

PARSER EXPECTED: <one of the following:
  a variable declaration, beginning with type,
      (int, real, vector, row_vector, matrix, unit_vector,
       simplex, ordered, positive_ordered,
       corr_matrix, cov_matrix,
       cholesky_corr, cholesky_cov
  or '}' to close variable declarations>
Error in stanc(file = file, model_code = model_code, model_name = model_name,  : 
  failed to parse Stan model '530b15df4662e9b168e0f021510d8a97' due to the above error.
```

Seems to fix if I add `rtsan` backend to the recompiling code (https://github.com/paul-buerkner/brms/blob/941057220d34e22db4ac9bd786cef557cedd1c1a/R/brmsfit-helpers.R#L762):
```
model_code = stancode(x, threads = threading(), backend = 'rstan'),
                                              ^^^^^^^^^^^^^^^^^^^
```
But I can submit that in different request? Or you check/change? Whatever's easiest